lancelot.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Raizel",
+			"oem": "Redmi",
+			"device": "Redmi 9",
+			"filename": "crDroidAndroid-13.0-20230911-lancelot-v9.8.zip",
+			"download": "",
+			"timestamp": 1694424716,
+			"md5": "25c990f24a6c3bb90367ee445ce80e9c",
+			"sha256": "05b2bccce77bf248b69abd815bd7b37946cc9387849e49807977572c336c3b5b",
+			"size": 1059155188,
+			"version": "9.8",
+			"buildtype": "Monthly",
+			"forum": "",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-T/16-Jul-2023/",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://mirrorbits.lineageos.org/full/lancelot/20230906/recovery.img",
```

Comment:
maybe use crDroid recovery?

Reply:
I tried that but seems it doesn't boot 
I will try to fix this later